File: alphabeticShift/alphabeticShift.ts
const alphabeticShift = (inputString: string): string => (
	
	inputString
		.split('')
		.map(char => char.charCodeAt(0) === 122 // charCode for 'z'
			? 'a' 
			: String.fromCharCode(char.charCodeAt(0)+1) )
		.join('') 
)

console.time('alphabeticShift x3')

console.log(alphabeticShift('crazy'));
console.log(alphabeticShift('bonkers'));
console.log(alphabeticShift('whattup'));

console.timeEnd('alphabeticShift x3')

File: alphabeticShift/alphabetic_shift.py

def alphabetic_shift(input_string: str) -> str:
    return ''.join(
        # return 'a' if the Unicode code point is for 'z' (122)
        chr(ord(char)+1) if ord(char) != 122 else 'a'
        for char in input_string
    )

print(alphabetic_shift('crazy'))
print(alphabetic_shift('bonkers'))
print(alphabetic_shift('whattup'))
